docs/bokeh/source/docs/dev_guide/writing_tests.rst

```diff
@@ -87,7 +87,7 @@ Python integration tests use pytest fixtures to handle the web driver
 configuration and interaction with Selenium. Depending on which context you
 want to test an object in, choose from ``bokeh_model_page``,
 ``single_plot_page``, or ``bokeh_server_page``. See
-:bokeh-tree:`bokeh/_testing/plugins/project.py` for more details.
```

Comment:
@bryevdv This path seems to have changed quite a bit - did I reference the correct file here?

Reply:
these were moved out of the package to the test dir: https://github.com/bokeh/bokeh/tree/branch-3.0/tests/support/plugins